src/api/goods.ts

```diff
@@ -1,11 +1,7 @@
 // eslint-disable-next-line
 const API_URL = `https://mate-academy.github.io/react_dynamic-list-of-goods/goods.json`;
 
-export function getAll(): Promise<Good[]> {
+export function getData(): Promise<Good[]> {
   return fetch(API_URL)
     .then(response => response.json());
 }
-
-export const get5First = () => {};
-
-export const getRedGoods = () => {};
```

Comment:
Why did you delete these functions? You can add logic there and make easier app component